libs/common/src/middleware/front-end-error-handler.test.ts

```diff
@@ -0,0 +1,47 @@
+import httpMocks from 'node-mocks-http';
+import { ErrorRequestHandler, Request, Response } from 'express';
+import { HttpStatusCode } from 'axios';
+import createHttpError from 'http-errors';
+import { getFrontEndErrorHandler } from './front-end-error-handler';
+
+describe('front-end-error-handler', () => {
+  describe('frontEndErrorHandler', () => {
+    const problemWithServiceTemplateLocation = 'problemWithServiceTemplateLocation';
+    let frontEndErrorHandler: ErrorRequestHandler;
+    let res: httpMocks.MockResponse<Response>;
+    let req: httpMocks.MockRequest<Request>;
+    let next: jest.Mock;
+    beforeEach(() => {
+      jest.resetAllMocks();
+      frontEndErrorHandler = getFrontEndErrorHandler(problemWithServiceTemplateLocation);
+      ({ res, req } = getHttpMocks());
```

Comment:
Can we just use `httpMocks.createMocks()` here rather than creating a `getHttpMocks` function that's only used in one place